### PR TITLE
e2e: outgoing cr + new UI in all e2e

### PR DESCRIPTION
**Pull request by churik**

### CODE REVIEW

File: test/appium/tests/medium/test_activity_center.py [hunk 1]
```diff
@@ -202,33 +93,56 @@ def prepare_devices(self):
...
+        self.profile_2.just_fyi("Check outgoing contact request and cancel it")
+        for indicator in (self.home_2.notifications_unread_badge, self.home_2.contact_new_badge):
+            if not indicator.is_element_displayed():
+                self.errors.append("Unread indicator on contacts tab or on activity center is not shown for outgoing CR!")
+        self.home_2.contacts_tab.click()
+        if self.home_2.pending_contact_request_text.text != '1':
+            self.errors.append("The amount of contact requests is not shown for outgoing CR!")
+        self.home_2.pending_contact_request_button.click()
+        from views.home_view import ActivityCenterElement
+        outgoing_cr = ActivityCenterElement(self.drivers[1], self.default_username_1)
```
Comment (yevh-berdnyk) [inline test/appium/tests/medium/test_activity_center.py]: Let's move this to the home_view.py creating a method like
`def get_activity_center_element(self, name):
    return ActivityCenterElement(self.driver, name)`
so here it's be like
```suggestion
        from views.home_view import ActivityCenterElement
        outgoing_cr = self.home_1.get_activity_center_element(name=self.default_username_1)
```

File: test/appium/tests/medium/test_activity_center.py [hunk 2]
```diff
@@ -202,33 +93,56 @@ def prepare_devices(self):
...
+        outgoing_cr.cancel_contact_request()
+        self.home_2.close_activity_centre.click()
+
+        self.profile_1.just_fyi("Check there is no unread indicator and pending CR")
+        self.device_1.chats_tab.click()
+        self.home_1.pending_contact_request_button.wait_for_invisibility_of_element(30)
+        self.home_1.notifications_unread_badge.wait_for_invisibility_of_element(30)
```
Comment (yevh-berdnyk) [inline test/appium/tests/medium/test_activity_center.py]: If these waits fail with timeout we'll never what's in self.errors. I'd suggest to wrap them in try/except and add a respective message to self.errors if these elements don't disapear

File: test/appium/views/home_view.py [hunk 3]
```diff
@@ -267,63 +270,68 @@ def get_chat_from_activity_center_view(self, chat_name):
...
         chat.public_key_edit_box.click()
         chat.public_key_edit_box.send_keys(public_key)
-        one_to_one_chat = self.get_chat_view()
-        chat.confirm()
-        # chat.confirm_until_presence_of_element(one_to_one_chat.chat_message_input)
-        if add_in_contacts and one_to_one_chat.add_to_contacts.is_element_displayed():
-            one_to_one_chat.add_to_contacts.click()
+        chat.view_profile_new_contact_button.click_until_presence_of_element(chat.profile_block_contact)
+        if remove_from_contacts and chat.profile_remove_from_contacts.is_element_displayed():
+            chat.profile_remove_from_contacts.click()
```
Comment (yevh-berdnyk) [inline test/appium/views/home_view.py:305]: Why do wee need a logic related to rejecting a CR inside a method which is adding a contact? I'd suggest  to split it and  have  2 different methods like add_contact and reject_contact

File: test/appium/tests/medium/test_activity_center.py (hunk 1 above)
Comment (yevh-berdnyk) [reply test/appium/tests/medium/test_activity_center.py]: Ah, this method already exists: `get_chat_from_activity_center_view`

File: test/appium/views/home_view.py (hunk 3 above)
Comment (churik) [reply test/appium/views/home_view.py:305]: It is temp and I suggest to leave it as it is before introduction of new interface of contact's profile, which is completely different

File: test/appium/tests/medium/test_activity_center.py (hunk 1 above)
Comment (churik) [reply test/appium/tests/medium/test_activity_center.py]: fixed

File: test/appium/tests/medium/test_activity_center.py (hunk 2 above)
Comment (churik) [reply test/appium/tests/medium/test_activity_center.py]: fixed, thanks

**Comment by churik** [comment]
@yevh-berdnyk all comments are addressed, would you mind to re-review?

File: test/appium/views/home_view.py (hunk 3 above)
Comment (yevh-berdnyk) [reply test/appium/views/home_view.py:305]: that makes sense

**Review by yevh-berdnyk — APPROVED** [review]

**Comment by yevh-berdnyk** [comment]
> @yevh-berdnyk all comments are addressed, would you mind to re-review?

Awesome, thanks!

File: test/appium/tests/medium/test_activity_center.py [hunk 4]
```diff
@@ -202,33 +93,59 @@ def prepare_devices(self):
...
-        [home.chats_tab.double_click() for home in [self.home_1, self.home_2]]
+        self.profile_2.just_fyi("Check outgoing contact request and cancel it")
+        for indicator in (self.home_2.notifications_unread_badge, self.home_2.contact_new_badge):
+            if not indicator.is_element_displayed():
+                self.errors.append("Unread indicator on contacts tab or on activity center is not shown for outgoing CR!")
+        self.home_2.contacts_tab.click()
+        if self.home_2.pending_contact_request_text.text != '1':
+            self.errors.append("The amount of contact requests is not shown for outgoing CR!")
+        self.home_2.pending_contact_request_button.click()
+        outgoing_cr = self.home_2.get_activity_center_element(self.default_username_1)
```
Comment (yevh-berdnyk) [inline test/appium/tests/medium/test_activity_center.py]: sorry, my bad: it should be 
```suggestion
        outgoing_cr = self.home_2.get_chat_from_activity_center_view(self.default_username_1)
```

**Pull request merged**